src/pagingscrollview/src/NIPagingScrollView.h

```diff
@@ -75,6 +75,14 @@ typedef enum {
 // of the view.
 @property (nonatomic) CGFloat pageInset;
 @property (nonatomic) NIPagingScrollViewType type; // Default: NIPagingScrollViewHorizontal
+// Controls layout for RTL.
+// For iOS versions 9 and higher it uses semanticContentAttribute to determine the layout direction.
```

Comment:
For iOS versions 9 and higher the initial value is determined based on the semanticContentAttribute property.